docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/pod-security-standards.md

```diff
@@ -2,67 +2,59 @@
 title: Pod Security Standards
 ---
 
-[Pod Security Standards (PSS)](https://kubernetes.io/docs/concepts/security/pod-security-standards/) define security restrictions for a broad set of workloads.  They replace [Pod Security Policies](https://kubernetes.io/docs/concepts/security/pod-security-policy/), which are now deprecated.
-
+[Pod Security Standards (PSS)](https://kubernetes.io/docs/concepts/security/pod-security-standards/) define security restrictions for a broad set of workloads. They replaced, the now removed, [Pod Security Policies](https://kubernetes.io/docs/concepts/security/pod-security-policy/) in Kubernetes v1.25 and above.
 
 ## Upgrade to Pod Security Standards
 
-Make sure that all Pod Security Policies have been migrated, which should result in equivalent security controls still being enforced (e.g. Pod Security Admission, Gatekeeper, Kyverno) and all PodSecurityPolicy objects removed from the cluster.
+Make sure that all Pod Security Policies have been migrated, which should result in equivalent security controls still being enforced (e.g. [Pod Security Admission](https://kubernetes.io/docs/concepts/security/pod-security-admission/), [Gatekeeper](https://github.com/open-policy-agent/gatekeeper), [Kubewarden](https://www.kubewarden.io/), [Kyverno](https://kyverno.io/), [NeuVector](https://neuvector.com/)) and all PodSecurityPolicy objects removed from the cluster.
 
 :::info important
 The new policy enforcement must be added before the PodSecurityPolicy objects are removed, or you risk creating a window of opportunity for privilege escalation within the cluster.
-
 :::
 
 ## Pod Security Admission Configuration Templates
 
 Rancher comes with two Pod Security Admission Configuration (PSAC) templates that you can assign to a cluster:
```

Comment:
In some places internally we use PSACT, but technically it's `kind: PodSecurityAdmissionConfiguration` when users go to create one. Imho I'd prefer something like _Pod Security Admission (PSA) configuration template_ or _PSA configuration template_ on first mention, then _template_ for short.